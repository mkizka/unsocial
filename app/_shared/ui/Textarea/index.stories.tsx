import type { Meta, StoryObj } from "@storybook/react";

import { Textarea } from ".";

const meta: Meta<typeof Textarea> = {
  component: Textarea,
  args: {
    placeholder: "Textarea",
  },
};
export default meta;

type Story = StoryObj<typeof Textarea>;

export const Default: Story = {};
