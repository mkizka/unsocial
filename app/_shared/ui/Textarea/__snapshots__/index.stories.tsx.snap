// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`_shared/ui/Textarea Default smoke-test 1`] = `
<textarea class="flex min-h-[80px] w-full rounded-md border border-input bg-background px-3 py-2 text-sm ring-offset-background placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50"
          placeholder="Textarea"
>
</textarea>
`;
